engine/include/cubos/engine/tools/scene_editor/plugin.hpp

```diff
@@ -0,0 +1,24 @@
+#pragma once
+
+#include <cubos/engine/cubos.hpp>
+#include <string>
```

Comment:
If you do move the `SceneInfo` resource to the `.cpp` file (read the other comment first), don't forget to also move the `<string>` include since it won't be necessary anymore.